menu.h

```diff
@@ -0,0 +1,103 @@
+#include <iostream>
+#include "utils.h"
+
+using namespace std;
+namespace menu
+{
+    void menu()
+    {
+        cout << "Welcome to the game!" << endl;
+        cout << "Please select an option:" << endl;
+        cout << "1. Start Game" << endl;
+        cout << "2. Settings" << endl;
+        cout << "3. Quit" << endl;
+        int textsize=16;
+        int weightsize=400;
+        int choice;
+        cin >> choice;
+
+        switch (choice)
+        {
+        case 1:
+        {
+            // code for starting the game
+            cout << "Starting the game..." << endl;
+            break;
+        }
+        case 2:
+        {
+            // code for settings
+            cout << "Opening settings..." << endl;
+            cout << "chouse a setting you want to change" << endl;
+            cout << "Please select an option:" << endl;
+            cout << "1. Text Size" << endl;
+            cout << "2. Font Weight" << endl;
+            cout << "3. Face Name" << endl;
+            int Setting_shoice;
+            cin >> Setting_shoice;
+            switch (Setting_shoice)
+            {
+            case 1:
+            {
+                cout << "Pick Number" << endl;
+                cin >> textsize;
+                utils::customize_text(textsize);
+                menu();
+                break;
+            }
+            case 2:
+            {
+                cout << "Pick Weight" << endl;
+                cout << "1. light" << endl;
+                cout << "2. SemiLight" << endl;
+                cout << "3. Normal" << endl;
+                cout << "4. SemiBold" << endl;
+                cout << "5. Bold" << endl;
+                cin >> weightsize;
+                if (weightsize == 1)
```

Comment:
Could also use `switch` statement here...

Reply:
you we could by i like if more :D